components/automate-chef-io/layouts/_default/data-api.html

```diff
@@ -19,7 +19,7 @@
   </head>
   <body>
     <ReDoc spec-url="/api-docs/all-apis.swagger.json"></ReDoc>
-    <script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/redoc/bundles/redoc.standalone.js"></script>
```

Comment:
Removing the `@next` tag loads 2.0.0-rc.23 rather than 2.0.0-rc.22. According to [the jsdelivr docs)[https://www.jsdelivr.com/features] it should be possible to do something like `@2.0` to guard against accidentally upgrading to the next major version but I couldn't get that to work.